pyomo/common/log.py

```diff
@@ -17,12 +17,11 @@
 #
 # Utility classes for working with the logger
 #
-
+import io
```

Comment:
I would prefer if you do `from io import TextIOBase, StringIO`.

Reply:
I was trying to keep the number of symbols in this namespace down (especially because they are not used frequently), but can change it if you feel very strongly about it.